plugins/javadoc/src/main/kotlin/javadoc/JavadocPageCreator.kt

```diff
@@ -43,20 +42,20 @@ open class JavadocPageCreator(
                 name = c.name.orEmpty(),
                 content = contentForClasslike(c),
                 dri = setOf(c.dri),
-                modifiers = listOfNotNull(c.visibility[jvm]?.name),
-                signature = signatureProvider.signature(c).nodeForJvm(jvm),
+                signature = signatureProvider.signature(c).nodeForJvm(jvm).asJavadocNode(),
```

Comment:
Can we extract this call chain `signatureProvider.signature(c).nodeForJvm(jvm).asJavadocNode()` somewhere?

Reply:
extracted